SgfDevs/Views/Shared/_Layout.cshtml

```diff
@@ -69,8 +69,8 @@
                     @*<li><a href="/events/">Events</a></li>*@
                     <li><a href="/groups">Groups</a></li>
                     <li><a href="/directory">Directory</a></li>
-                    <li><a href="/jobs">Jobs</a></li>
-                    <li><a href="/archives">Archives</a></li>
+                    @* <li><a href="/jobs">Jobs</a></li> *@
```

Comment:
Might as well remove this since the mobile version was also removed

Reply:
How do you mean? I thought we originally wanted to keep this commented out so if we wanted to reimplement it it would go here. 

Up to you.